multiqc/modules/fastqc/assets/js/multiqc_fastqc.js

```diff
@@ -24,12 +24,20 @@ function load_fastqc_passfails() {
         var key_value = JSON.parse(elem.innerHTML);
         fastqc_passfails[key_value[0]] = key_value[1];
     });
+
+    // Backward compatible global variables
```

Comment:
Last commit can be removed if there's no need to keep backward compatibility of global javascript variables

Reply:
These files are basically always bundled together into a single HTML file, so I don't think that there is any need for backwards compatibility, no 👌